Frontend/ui-driver/src/Screens/EditBankDetailsScreen/View.purs

```diff
@@ -112,7 +112,7 @@ headerLayout state push heading =
         , height WRAP_CONTENT
         , text (getString EDIT)
         , margin (MarginRight 10)
-        , color Color.blueBtn
+        , color Color.brightBlue
```

Comment:
This screen is not being used, Why the changes is here ?

Reply:
Color name is changed